prac_09/taxi_test.py

```diff
@@ -0,0 +1,30 @@
+"""
+CP1404/CP5632 Practical
+Taxi Test Program
+"""
```

Comment:
Time estimate?
Also nice bit of code, i didn't think to create a test program for this. Good work